### DISCUSSION TITLE
Theme Organizer board or prospect Kanban

### DISCUSSION THREAD

**Discussion by olivier-nolf** [mokhosh/filament-kanban#35] [Q&A] [answered]
Hi, 

Do you also have the themes/blades for the organizer board or prospect kanban as in your screenshots?
I like the layout and it would same a lot of time on styling everything.

Thanks & Kr,
Olivier

**Comment by mokhosh** [comment, marked as answer]
Hey,

Have you looked at the `kanban-todo` repository that I made for this? I have basically implemented the Prospect theme that you're referring to in that example.
Here are the views:
https://github.com/mokhosh/filament-kanban-todo/tree/main/resources/views/tasks-kanban

And in these videos I go over the details:
https://www.youtube.com/watch?v=RF6-2hern08
https://www.youtube.com/watch?v=hyhmEIoWqTg
https://www.youtube.com/watch?v=WddCaqyE0D0

Let me know if these are not enough, and I will try to help further.
